src/components/Icon/Icon.tsx

```diff
@@ -200,12 +206,15 @@ function Icon({
   );
 
   let contentMarkup: React.ReactNode;
-
   if (source === 'placeholder') {
-    contentMarkup = <div className={styles.Placeholder} />;
+    contentMarkup = <div testID="placeholder" className={styles.Placeholder} />;
```

Comment:
I don't think we need to use a testID in this situation, we'll only have one div if any
```suggestion
    contentMarkup = <div className={styles.Placeholder} />;
```